### ISSUE TITLE
Game doesn't launch without any error

### ISSUE THREAD

**Issue by mahdisml** [stride3d/stride#1996]
**Release Type**: Official Release

**Version**: 4.1

**Platform(s)**: Windows

 - Device: PC
 - OS: Windows
 - Version 11
 - CPU Intel 6200U
 - GPU NVIDIA 940M , intel HD Graphics 520
 - Memory 12GB

**Describe the bug**
project build successfully but it doesn't run in editor

**To Reproduce**
1. start a new project (any stater sample)
2. click run button

**Expected behavior**
running game

**Screenshots**
![image](https://github.com/stride3d/stride/assets/1758504/00550722-1d39-4ee8-a1ea-8c870f4eba40)

**Log and callstacks**
If any, please attach here any log or callstack (preferably in a `.txt` file using GitHub drag and drop)

**Additional context**
[logg.txt](https://github.com/stride3d/stride/files/13194650/logg.txt)

**Comment by manio143** [comment]
Can you run `eventvwr` and see if any error is reported?
![image](https://github.com/stride3d/stride/assets/10709060/90916cb4-53f6-4326-8c34-52b6aa03cab5)

**Comment by mahdisml** [comment]
> Can you run `eventvwr` and see if any error is reported? ![image](https://user-images.githubusercontent.com/10709060/279199828-90916cb4-53f6-4326-8c34-52b6aa03cab5.png)

I checked , there is no error.

**Comment by manio143** [comment]
Can you see if it runs from VS or with `dotnet run`?

**Comment by mahdisml** [comment]
> Can you see if it runs from VS or with `dotnet run`?

i got an error :

```
Unhandled exception. System.IO.FileNotFoundException: Could not load file or assembly 'System.Windows.Forms, Version=6.0.2.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'. The system cannot find the file specified.
File name: 'System.Windows.Forms, Version=6.0.2.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'
   at Stride.Games.GameContextFactory.NewGameContextDesktop(Int32 requestedWidth, Int32 requestedHeight, Boolean isUserManagingRun)
   at Stride.Games.GameContextFactory.NewGameContext(AppContextType type, Int32 requestedWidth, Int32 requestedHeight, Boolean isUserManagingRun)
   at Stride.Games.GameBase.Run(GameContext gameContext)
   at FirstPersonShooter.FirstPersonShooterApp.Main(String[] args) in C:\labs\strd\FirstPersonShooter\FirstPersonShooter.Windows\FirstPersonShooterApp.cs:line 11

Process finished with exit code -532,462,766.

```

**Comment by manio143** [comment]
What version of .NET 6 SDK do you have installed? You may need to update it. 6.0.2 SDK had apparently issues as per https://github.com/dotnet/winforms/issues/6663

**Comment by mahdisml** [comment]
It fixed with 4.2 Version ( .net8 )
Thanks guys !
